### PR TITLE
Update on 2021 SSI and TANF data

### PR DESCRIPTION
**Pull request by Weishan-Lu**
Thanks for contributing! Please remove any top-level sections that do not apply to your changes.

- [ ] `make format && make documentation` has been run.

# New variable

- [ ] Label field added
- [ ] Documentation field added
- [ ] Unit field added
- [ ] Default value field added if relevant
- [ ] Variable name follows conventions
- [ ] Unit test(s) added
- [ ] Integration test(s) added if relevant
- [ ] Issues this PR fixes linked

## What's changed

Description of the changes here.

# Bug fix

- [ ] Regression test added
- [ ] Regression test passing

## What this fixes and how it's fixed

Description of how this fix works goes here. Link any issues this PR fixes.

### CODE REVIEW

File: policyengine_us/parameters/calibration/SSI/payments.yaml [hunk 1]
```diff
@@ -0,0 +1,112 @@
+description: Total amount of federally administered SSI payments for 2021 by federal and state
+metadata: 
+  unit: currency-USD (in thousands of dollars)
```
Comment (nikhilwoodruff) [inline policyengine_us/parameters/calibration/SSI/payments.yaml:3]:
```suggestion
  unit: currency-USD
```

Could we also make the numbers 1-1?

File: policyengine_us/parameters/calibration/SSI/recipients.yaml [hunk 2]
```diff
@@ -0,0 +1,112 @@
+description: Total number of recipients of federally administered SSI payments for December 2021 by federal and state
+metadata: 
+  unit: person
+  period: December 2021
+  reference:
+  - title: Supplemental Security Income State Data | Social Security Administration
+    href: https://www.ssa.gov/policy/docs/statcomps/supplement/2022/7b.html#table7.b1
+Federal:
```
Comment (nikhilwoodruff) [inline policyengine_us/parameters/calibration/SSI/recipients.yaml:8]: Same as above

**Review by nikhilwoodruff — APPROVED** [review]
Good to merge after the suggested changes- thanks!

**Comment by MaxGhenis** [comment]
Could we make all folders and files lowercase to match parameters and variables?

**Comment by nikhilwoodruff** [comment]
Yes + final comment: could we add an entry to `changelog_entry.yaml` (see recently merged PRs for examples) with the description of this PR and a `patch` update?

File: changelog.yaml [hunk 3]
```diff
@@ -2975,3 +2975,8 @@
     changed:
     - CalEITC reimplemented without branching and without bug.
   date: 2023-04-14 10:05:46
+- bump: patch
```
Comment (MaxGhenis) [inline changelog.yaml:2978]: This content goes in changelog_entry.yaml, not changelog.yaml. You can see other recently merged PRs for examples.

**Pull request merged**